crates/solidity/testing/perf/src/lib.rs

```diff
@@ -19,6 +19,12 @@ mod unit_tests {
                 crate::tests::$name::run(payload);
             }
         };
+        ($name:ident, empty_payload) => {
```

Comment:
super nit: WDYT of a different macro, instead of the special `empty_payload` identifier?

```rust
macro_rules! define_test {}
macro_rules! define_payload_test {}
```

Reply:
Sounds good, I'll change it.